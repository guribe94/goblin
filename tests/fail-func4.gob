world[20,40] {

}
entities {
  @:player {
    build {
    }
    does {
      test(this);
      exit;
    }
  }

}
functions {

  num foo() {}

  bool bar() {}

  char getKey() {} /* Should not be able to define built-in */

  bool baz() {}

  bool test(entity p) {

    return true;
  }
}
