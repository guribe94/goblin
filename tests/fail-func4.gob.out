Fatal error: exception Failure("function getKey may not be defined")
